### PR TITLE
Add Sequel adapter

### PR DESCRIPTION
**Pull request by ssnickolay**

### CODE REVIEW

File: lib/clowne/adapters/active_record/associations/base.rb [hunk 1]
```diff
@@ -48,6 +48,10 @@ def with_scope
 
           private
 
+          def clone_record(record)
+            Clowne.resolve_adapter(:active_record).copier.call(record)
```
Comment (palkan) [inline lib/clowne/adapters/active_record/associations/base.rb]: Frankly speaking, I don't think it make sense to separate _pure_ cloning logic from adapter.
If you need custom _copier_ then just add another adapter (e.g. `:custom_active_record`).

Comment (palkan) [reply lib/clowne/adapters/active_record/associations/base.rb]: Or even better: add specific declaration (`initialize_with ->(source) { ... }`) to the configuration to make it possible to re-define default adapter behaviour in-place.

Anyway, let's postpone this change.

File: lib/clowne/adapters/sequel.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+module Clowne
+  module Adapters
+    # Cloning adapter for Sequel
+    class Sequel < Base
+      # Adds #cloner_class method to Sequel::Model
+      module SequelExt
+        def cloner_class
+          return @_clowne_cloner if instance_variable_defined?(:@_clowne_cloner)
+
+          cloner = "#{name}Cloner".safe_constantize
```
Comment (palkan) [inline lib/clowne/adapters/sequel.rb]: We cannot use `safe_constantize` here, 'cause it's from ActiveSupport and Sequel doesn't depend on it.

We already have `StringConstantize` refinement with `#constantize` method. We can enhance and use it.

Btw, maybe, we can share this part with AR? It looks identical.

Comment (ssnickolay) [reply lib/clowne/adapters/sequel.rb]: Damn, you're right! 
Fixed and implemented ORM extension https://github.com/palkan/clowne/pull/4/commits/db9dd4c6bbb7cb02eb2d9acd165192be48201306

File: lib/clowne/adapters/active_record/associations/base.rb (hunk 1 above)
Comment (ssnickolay) [reply lib/clowne/adapters/active_record/associations/base.rb]: Answered in a email ;)

Comment (ssnickolay) [reply lib/clowne/adapters/active_record/associations/base.rb]: @palkan https://github.com/palkan/clowne/pull/4/commits/656089891a45bbc2318a5a1fa66b1cd2af6dbe9c

**Comment by ssnickolay** [comment]
@palkan done 🔝

File: .codeclimate.yml [hunk 3]
```diff
@@ -1,3 +1,6 @@
+engines:
+  rubocop:
+    enabled: true
```
Comment (palkan) [inline .codeclimate.yml]: We already run RuboCop as a part of our default Rake task in Travis, I don't think we need it here

**Pull request merged**